app/templates/components/component-tree-item.hbs

```diff
@@ -21,10 +20,81 @@
   {{/if}}
 
   <code
-    class="component-tree-item__tag {{if @item.isComponent "component-tree-item__bracket"}} {{if (not @item.hasChildren) "component-tree-item__bracket--self-closing"}}"
+    class="component-tree-item__tag
+      {{if
+        (and (not @item.isCurlyInvocation) @item.isComponent)
+        "component-tree-item__bracket"
+      }}
+      {{if
+        (and (not @item.isCurlyInvocation) (not @item.hasChildren) @item.isComponent)
+        "component-tree-item__bracket--self-closing"
+      }}
+      {{if
+        (and @item.isCurlyInvocation @item.isComponent)
+        "component-tree-item-classic__bracket"
+      }}"
   >
     {{#if @item.isComponent}}
-      {{classify @item.name}}
+      {{#if @item.args.positional}}
+        <span class="component-name">
+          {{@item.name}}
+        </span>
+
+        {{#each @item.args.positional as |value|}}
+          <div class="arg-token">
+            <span class="value-token">
+              {{component-argument value}}
+            </span>
+          </div>
+        {{/each}}
+
+        {{#each-in @item.args.named as |name value|}}
+          <div class="arg-token">
+            <span class="key-token">
+              {{name}}
+            </span>
+            =
+            <span class="value-token">
+              {{component-argument value}}
+            </span>
+          </div>
+        {{/each-in}}
+      {{else}}
+        <span class="component-name">
+          {{classify @item.name}}
+        </span>
+
+        {{#each-in @item.args.named as |name value|}}
+          <div class="arg-token">
+            {{#if (is-string value)}}
```

Comment:
Instead of breaking up and duplicating the whole block, I think you can just add something like...

```hbs
{{#if (is-string value)}}"{{else}}<span class="bracket-token">\{{</span>{{/if}}
<span class="value-token">
  {{component-argument value}}
</span>
{{#if (is-string value)}}"{{else}}<span class="bracket-token">}}</span>{{/if}}
```

...around the values, similar to what I showed above.